common/quotas/ratelimiter.go

```diff
@@ -58,8 +58,9 @@ type RateLimiter struct {
 }
 
 // NewSimpleRateLimiter returns a new rate limiter backed by the golang rate
-// limiter
-func NewSimpleRateLimiter(rps int) *RateLimiter {
+// limiter.  This is currently only used in tests.
+func NewSimpleRateLimiter(t *testing.T, rps int) *RateLimiter {
+	t.Helper() // ensure a T has been passed
```

Comment:
seems very likely that this will only ever be useful in tests

Reply:
I'd consider s/Simple/Test/ because it presumably means this has testing functionality